py/desisim/scripts/quickbrick.py

```diff
@@ -228,8 +233,9 @@ def main(args):
     meta.add_column(Column(true_objtype, dtype=(str, 10), name='TRUE_OBJTYPE'))
     meta.add_column(Column(targetids, name='TARGETID'))
 
+    # @sbailey asks: What was this supposed to do for OBJTYPE vs. TRUE_OBJTYPE?
     # Now add fixed-up OBJTYPE to the template meta table
```

Comment:
Previously ```OBJTYPE``` was not included in the metadata table returned by ```desisim.templates```, but it was added as part of #132, which broke this part of the code.  We need this renaming of ```OBJTYPE--->TRUE_OBJTYPE``` for contaminants, however, for example, to identify objects with ```OBJTYPE='QSO_BAD'``` which are actually ```TRUE_OBJTYPE='STAR'```.  (Note that I didn't write this particular part of ```quickbrick``` but I believe that it's still needed downstream in the redshift fitting and classification portion of the pipeline.)

Long story short I think the line that's commented out can be removed safely.